plugins/oauth/hooks.js

```diff
@@ -201,8 +201,9 @@ module.exports = Object.assign({}, StoreMixin, {
 
   userLoginSuccess() {
     const redirectTo = RouterUtil.getRedirectTo();
+    const isValidRedirect = RouterUtil.isValidRedirect(redirectTo);
```

Comment:
Correct. We need to pass the entire domain/url that was extracted from redirect query